src/opnsense/mvc/app/controllers/OPNsense/Unbound/forms/dnsbl.xml

```diff
@@ -41,7 +41,22 @@
         <type>select_multiple</type>
         <style>tokenize</style>
         <allownew>true</allownew>
-        <help>List of domains to blocklist. Only exact matches are supported.</help>
+        <help>
+            List of domains to blocklist. Only exact matches and wildcards are supported.
```

Comment:
Wait, this help text is wrong now with nbound.dnsbl.wildcards addition, right?

Reply:
You're right, missed a spot. Thanks:)